pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.mozilla</groupId>
+    <artifactId>jss</artifactId>
+    <version>4.8.0-SNAPSHOT</version>
+
```

Comment:
Apparently we need this to start compiling native sources:
```
<packaging>so</packaging>
```

Reply:
Ah yuck. So we've gotta split JSS into two maven projects and build them both separately. Gotcha.